web/packages/teleport/src/services/sales/index.ts

```diff
@@ -44,6 +44,7 @@ export function getSalesURL(
 ) {
   let url = UPGRADE_COMMUNITY_URL;
   if (isEnterprise) {
+    // TODO(mcbattirola): use a different field to set the URL or add new URLs for new products
```

Comment:
Nit - can we have a consistent comment for replacing the remaining isTeam references?